server/apps/main/templates/main/signup.html

```diff
@@ -0,0 +1,125 @@
+
+    <script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.0/jquery.min.js'></script>
+
+
+    <!-- Bootstrap -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+    <title>Sign Up</title>
+    <style>
+        body {
+            height: 100vh;
+        }
+        .bg-light {
+            background-color:rgba(149, 239, 234, 0.42);
+            margin-bottom: 0;
+        }
+
+        ul {
+            list-style-type: none;
+            width: 100%;
+            display: flex;
+            justify-content: center;
+        }
+        .nav-item {
+            padding: 1.5em 2em 1.5em 2em;
+            font-size: large;
+            font-weight: bold;
+        }
+        .nav-link {
+            color: black;
+        }
+
+        .nav-link:hover {
+            text-decoration: none;
+            color: black;
+        }
+        #project-sign-up {
+            margin-left: 2em;
+            font-size: large;
+            margin-top: 0.5em;
+        }
+        .container {
+            margin: auto;
+        }
+        #form-check {
+            display: flex;
+            /* height: 50vh; */
+            padding-top: 25vh;
+
+        }
+        .footer {
+            background-color: gray;
+            height: 10vh;
+        }
+        .submit {
+            margin-left: 45%;
+        }
+        input[type="radio"]{
+           margin: 0 10px 0 10px;
+        }
+
+        #question p {
+            font-size: large;
+        }
+    </style>
+</head>
+<body>
+    <nav class="navbar fixed-top navbar-light bg-light">
+        <ul class="navbar-nav justify-content-center">
+            <li class="nav-item">
+              <a class="nav-link" href="#">ABOUT</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Share</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">View</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">MyPage</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">LOGIN</a>
+            </li>
+          </ul>
+    </nav>
+    <div id="project-sign-up">
+        <p>Sign Up to AutSPACE</p>
+    </div>
+    <form id="form-check" action="{% url 'signup_frame4_test' %}" method="POST">
```

Comment:
You're missing `main:` before `signup_frame4_test`. 

Otherwise, I get an error during template rendering. 
![CleanShot 2021-07-07 at 12 49 37](https://user-images.githubusercontent.com/1872836/124798808-cfe8fe00-df21-11eb-8776-bd49240514ce.jpg)